src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -475,8 +476,8 @@ public async Task<IActionResult> Edit(WorkflowTypeUpdateModel model)
             foreach (var activityState in activities)
             {
                 var activity = currentActivities[activityState["id"].ToString()];
-                activity.X = (int)Convert.ToDecimal(activityState["x"].ToString());
-                activity.Y = (int)Convert.ToDecimal(activityState["y"].ToString());
+                activity.X = (int)Convert.ToDecimal(activityState["x"].ToString(), CultureInfo.InvariantCulture);
```

Comment:
Since this statically configured the culture, it would allow only a decimal dot, not a decimal comma. I don't think that's necessarily a suitable solution for everyone.

This can be the storage format, but then depending on the UI culture, the input needs to be changed before saving.

Reply:
The issue is that Convert.ToDecimal uses the culture info when parsing the internal value. It's never entered manually.

Reply:
Oh OK then!